modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PersistentPageMemory.java

```diff
@@ -282,6 +285,14 @@ public PersistentPageMemory(
         }
 
         delayedPageReplacementTracker = new DelayedPageReplacementTracker(pageSize, flushDirtyPageForReplacement, LOG, sizes.length - 1);
+
+        //        new PagesWriteThrottle(
```

Comment:
Are these comments necessary? Or write why it is needs.

Reply:
This is the initialization code for throttling. It is commented because throttling is disabled in main. I will write a comment about what it is